.github/workflows/test.yml

```
@@ -0,0 +1,27 @@
+name: Run tests
+
+on:
+  push:
+    branches: [ master, main, nightly ]
+  pull_request:
+    branches: [ master, main, nightly ]
```

@jfavellar90 can this be run for all PRs?

`
```suggestion
    branches: *
```

---

@Ian2012 for now I'm following tutor [strategy](https://github.com/overhangio/tutor/blob/v17.0.4/.github/workflows/test.yml#L7)